src/Common/ORDataObject/ORDataObject.php

```diff
@@ -76,7 +78,7 @@ function populate()
         }
     }
 
-    function populate_array($results)
+    protected function populate_array($results)
```

Comment:
I guess without Camel Caps Travis will be angry on me 😕

Reply:
don't worry about that. we have requested travis to not get angry about that :)
https://github.com/openemr/openemr/blob/master/ci/phpcs.xml#L8

Reply:
Done with my changes but still, Travis is angry 😥